crates/ln-dlc-node/src/tests/just_in_time_channel/create.rs

```diff
@@ -123,6 +127,17 @@ pub(crate) async fn send_interceptable_payment(
 
     payer.send_payment(&invoice)?;
 
+    if TestPath::MobileFunding == test_path {
+        // simulate the user switching from another app to 10101
+
+        // Note, hopefully Breez, Phoenix or any other non-custodial wallet is able to run in the
+        // background when sending a payment as otherwise a disconnect on their side would happen
+        // when switching to 10101 resulting into a failed payment.
+        // payer.disconnect(coordinator.info);
```

Comment:
did you mean to disconnect the payer here? 

the line seems relevant, but it's commented out.

Reply:
I guess the comment above could be more clear. I did comment it that because it wouldn't work otherwise.

Reply:
perhaps:
`// line below is commented out on purpose`

or actually remove the commented out line of code and just keep the comment?